src/sentry/static/sentry/app/views/projectEnvironments.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Reflux from 'reflux';
+import createReactClass from 'create-react-class';
+
+import EnvironmentStore from '../stores/environmentStore.jsx';
+
+import Panel from './settings/components/panel';
+import PanelHeader from './settings/components/panelHeader';
+import PanelBody from './settings/components/panelBody';
+import EmptyMessage from './settings/components/emptyMessage';
+import {t} from '../locale';
+
+import Row from './settings/components/row';
+import SettingsPageHeader from './settings/components/settingsPageHeader';
+import ListLink from '../components/listLink';
+
+const ProjectEnvironments = createReactClass({
+  mixins: [Reflux.listenTo(EnvironmentStore, 'onEnvironmentsChange')],
```

Comment:
Do we use `EnvironmentStore` elsewhere? If so, a HoC would be nice, especially with planned redux change.

Reply:
👍 Will open a PR for this